src/CFamily.UnitTests/Rules/DynamicCFamilyRulesConfigTests.cs

```diff
@@ -150,30 +148,6 @@ public void ActiveRules_CustomSettingsOverrideDefaults()
             testSubject.ActivePartialRuleKeys.Should().BeEquivalentTo("rule1", "rule3");
         }
 
-        [TestMethod]
-        public void ActiveRules_ExcludedRulesOverrideCustomAndDefaults()
```

Comment:
Deleted all tests relating to excluded rules. There are new tests in `RulesConfigFixupTests.cs`.